src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,61 @@
+package core.basesyntax;
+
+import core.basesyntax.figures.Circle;
+import core.basesyntax.figures.IsoscelesTrapezoid;
+import core.basesyntax.figures.Rectangle;
+import core.basesyntax.figures.RightTriangle;
+import core.basesyntax.figures.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final String DEFAULT_COLOR = "WHITE";
+    private static final int DEFAULT_RADIUS = 10;
+    private static final int QUANITY_OF_TYPES = 5;
+    private static final int CONSTANT = 100;
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private String color = colorSupplier.getRandomColor();
+    private Random random = new Random();
+    private int figureType = random.nextInt(QUANITY_OF_TYPES);
```

Comment:
same for figureType, it cannot be generated ones, move inside getRandomFigure() method before switch oeprator